cmd/memorydiskserver/memorydiskserver.go

```diff
@@ -63,6 +75,15 @@ func setHandler(w http.ResponseWriter, r *http.Request) {
 //}
 
 func main() {
+	// Read the starting value
+	dat, err := ioutil.ReadFile(STORE)
```

Comment:
I think this can be simplified to:

if dat, err := os.Create(STORE); err != nil {
    // couldn't even create the file, something is very wrong
}
defer f.Close()
var data []byte
if _, err := dat.Read(data); err != nil {
    // couldn't read the data, something is really wrong
}

Doesn't exactly shorten the code but makes it a bit more readable (I think)